### PR TITLE
added comment functionality

### PR DESCRIPTION
**Pull request by HeyNitin**
Added comment functionality

https://deploy-preview-7--strivesocial.netlify.app/

### CODE REVIEW

File: src/pages/postPage/postPage.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    const postComment = async () => {
+        try {
+            const res = await axios.post(`/api/comments/add/${postId}`, { commentData: textArea }, { headers: { authorization: token } })
+            Dispatch(setPosts(res.data.posts))
+            setTextArea('')
+        }
+        catch (error) {
+            showToast('error', "Something went wrong while trying to post your comment")
+        }
+    }
```
Comment (partha8) [inline src/pages/postPage/postPage.tsx:45]: Suggestion: Why not place the login in slice?

**Review by partha8 — APPROVED** [review]
image

The UI is breaking on a bigger screen.
The functionality works, and the code is clean and readable as well.

File: src/pages/postPage/postPage.tsx (hunk 1 above)
Comment (HeyNitin) [inline src/pages/postPage/postPage.tsx:45]: which login exactly? user auth is in a userSlice

**Comment by HeyNitin** [comment]
> image
> 
> The UI is breaking on a bigger screen. The functionality works, and the code is clean and readable as well.

Yes, I'm aware of it. I'll make it responsive, once I finish all functionalities

File: src/pages/postPage/postPage.tsx (hunk 1 above)
Comment (partha8) [reply src/pages/postPage/postPage.tsx:45]: my bad, I mixed other's review here.

**Comment by shivamSspirit** [comment]
ui collapse

**Pull request merged**